src/pages/home/components/HowItWorksSection/HowItWorksSection.tsx

```diff
@@ -0,0 +1,84 @@
+import { motion } from "framer-motion"
+import Heading from "@/components/ui/Heading"
+import customizePlaylistImage from "@public/customize-playlist.svg"
+import spotifyLockImage from "@public/spotify-lock.svg"
+import searchArtistsImage from "@public/search-artists.svg"
+import AnimatedSection from "./AnimatedSection"
+
+const containerVariants = {
```

Comment:
I see `containerVariants` and `itemVariants` are repeated across files. Should we reuse them?

Reply:
I think it has different values inside, but I can think of reusing this code.

Reply:
I have created a shared file for the framer motion variants in the lib folder. It looks great 👌